web/themes/custom/votegov/src/sass/components/to-top.scss

```diff
@@ -0,0 +1,57 @@
+@use "uswds-core" as *;
+@use "mixins" as *;
+@use "variables" as *;
+
+.back-to-top {
+  --vote-button-background: #{$base-white};
+  --vote-button-border: #{$base-alt-hover};
+  --vote-button-arrow-icon: #{$ac-dark};
+
+  [data-theme="contrast"] & {
+    --vote-button-border: #{$link-high-contrast};
+  }
+
+  @include hover {
+    --vote-button-background: #{$base-alt-hover};
+    --vote-button-arrow-icon: #{$base-white};
+
+    [data-theme="contrast"] & {
+      --vote-button-background: #{$link-high-contrast};
+      --vote-button-arrow-icon: #{$base-white};
+    }
+  }
+
+  background-color: var(--vote-button-background);
+  border: 2px solid var(--vote-button-border);
+  border-radius: 50%;
+  outline: 1px solid $base-white;
+  width: 48px;
+  height: 48px;
+  position: fixed;
+  bottom: 1.25rem;
+  right: 2rem;
+  margin: auto;
+  transition: all 200ms;
+  opacity: 0;
+  visibility: hidden;
+  z-index: 1000;
+
+  @include at-media('tablet') {
+    bottom: 3.5rem;
+  }
```

Comment:
Can you replace whatever you can with uswds mixins like `@include u-margin('auto');`